pkg/pb/inspector.proto

```diff
@@ -184,10 +184,11 @@ message DashboardResponse {
   string bootstrap_address = 3;
   string internal_address = 4;
   string external_address = 5;
-  StatSummaryResponse stats = 6;
-  google.protobuf.Duration uptime = 7;
-  google.protobuf.Timestamp last_pinged = 8;
-  google.protobuf.Timestamp last_queried = 9;
+  string dashboard_address = 6;
```

Comment:
I suppose that rearranging fields isn't currently a problem, because we can introduce breaking changes.